front/types/shared/utils/string_utils.ts

```diff
@@ -122,6 +122,7 @@ export function asDisplayName(name?: string | null) {
   return name
     .toLowerCase()
     .replace(/_/g, " ")
+    .replace(/github/g, "GitHub")
```

Comment:
Am I allowed to add this special case here?

Reply:
:see_no_evil: